cli/ssh_client.py

```diff
@@ -110,7 +110,9 @@ def ssh(self, cmds, host, output=None):
         parts = cmd.split(' ')
         parts.append(' && '.join(cmds))
         CONSOLE.debug(json.dumps(parts))
-        ret_val = subprocess_to_log.call(parts, LOG, log_id=host, output=output, scan_for_errors=[r'lost connection', r'\s*Failed:\s*[1-9].*'])
+        ret_val = subprocess_to_log.call(parts, LOG, log_id=host, output=output, scan_for_errors=[r'lost connection',
+                                                                                                  r'\s*Failed:\s*[1-9].*a',
+                                                                                                  r'.*NameError:\s*.*'])
```

Comment:
Looking for NameError is catching one specific error condition, it would be better to look for something more general otherwise the next error that is slightly different won't be caught. Looking at the log I suggest "Failures:".

Reply:
Sure, James. Thanks.